### DISCUSSION TITLE
v2.10.2  7953752

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1479] [General]
Error: File"artisanlib\pid_dialogs.py", line 847, in okAction
https://github.com/artisan-roaster-scope/artisan/assets/149906183/470b988c-04fa-4c54-abcb-9fb33f3eb966

**Comment by Terracotta-6** [comment]
continuous  ecc6b1c

![4b7e3599e5ca661082d976799af0e5b](https://github.com/artisan-roaster-scope/artisan/assets/149906183/a5a3a427-c748-4c16-8a1c-e5cef015bee8)

**Reply by Terracotta-6** [reply]
![4dc18bbf15095ce1dea31ce91d096d1](https://github.com/artisan-roaster-scope/artisan/assets/149906183/743256aa-52a7-4cd2-b1e6-d818cea2883d)

**Comment by Terracotta-6** [comment]
[artisan-settings.aset.txt](https://github.com/artisan-roaster-scope/artisan/files/14471205/artisan-settings.aset.txt)

**Comment by MAKOMO** [comment]
My extensions of the internal software PID for v2.10.2 broke the PID dialog for the other options (TC4/Kaleido, MODBUS, S7). Sorry! Commit 4ae5c56f09e54606cbc7d677853452e39db6e30e should fix this. Could you please verify using the current Continuous build works better for you?My extensions of the internal software PID for v2.10.2 broke the PID dialog for the other options (TC4/Kaleido, MODBUS, S7). Sorry! Commit 4ae5c56f09e54606cbc7d677853452e39db6e30e should fix this. Could you please verify using the current Continuous build works better for you?

**Reply by Terracotta-6** [reply]
Commit 4ae5c56 This issue has been fixed.

**Reply by Terracotta-6** [reply]
v2.10.2's internal software PID control is very interested in testing the functionality.

**Comment by Terracotta-6** [comment]
![bcfba30f15943c63f99a1469b8d2102](https://github.com/artisan-roaster-scope/artisan/assets/149906183/f5646018-b27b-4a69-92d0-f27a4b009311)
